import-service/serverless.yml

```diff
@@ -77,6 +77,12 @@ functions:
           path: /import
           method: get
           cors: true
+          authorizer:
+            name: basicAuthorizer
+            arn: arn:aws:lambda:eu-west-1:376875946089:function:authorization-service-dev-basicAuthorizer
```

Comment:
Will it be changed during each auth-service deployment? Is so, then it's better to use this:
![image](https://user-images.githubusercontent.com/33042066/100789408-75d20880-3427-11eb-85c6-8797beb4df81.png)